devtool

```diff
@@ -176,7 +176,7 @@ install_deps_dev() {
 
 install_package() {
     wheel_name=$(ls -t dist | head -n 1)
-    pip3 install --upgrade dist/$wheel_name
+    pip3 install --upgrade dist/$wheel_name --upgrade --upgrade-strategy only-if-needed --force-reinstall
```

Comment:
Aren't `--upgrade-strategy only-if-needed` and `--force-reinstall` opposing ideas?

Reply:
Thanks! You're right `--upgrade-strategy only-if-needed` is the default option which is not needed here. I'll delete in another commit.